### ISSUE TITLE
How to load (require) a specific version of a ruby gem?

### ISSUE THREAD

**Issue by stavnstrup** [Compass/compass#1416]
In the documentation the require statement is used to load a library of code. So the config.rb file should e.g. contain the statement

require 'susy'

Documentation is missing for instructions on how to specify a specific version number, which is necessary, if we have multiple version of a gems installed?

Note, that this requirement is for a simple website (not Rails)

**Comment by mirisuzanne** [comment]
There's no way to set requirement versions in the compass config, but you can do it using tools like [bundler](http://bundler.io/), and [RVM](https://rvm.io/). See [Susy's version management documentation](http://susy.oddbird.net/guides/getting-started/#troubleshooting-versions) for more.

**Comment by stavnstrup** [comment]
After I reported this issue, I found this page

  http://guides.rubygems.org/patterns/

which seems to indicate, that it is actually possible. I have tried it with the following statements in the config.rb file

   gem 'zurb-foundation', '=3.2.5'
   require 'zurb-foundation'

and it seems to work. However, I can not be sure, since I do not know, how to debug Compass and the trace options does not provide any data.

I am sure Bundler and RVM are the appropiate way for Ruby on Rails apps, but the Susy page mentions that for a pure Compass install, I should uninstall the old version and install the new one. That is not an option, as I would like to use different versions of Zurb Foundation for different sets of SASS files.
